README.md

```diff
@@ -1 +1,68 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> This is first solo project for learnig purpose 
+
+
+## Built With
+
+- HTML
+- VS Code
+
+## Getting Started
+
+-I have just created repository in Github and clone it in my local machine to further changes
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  -github basic commands
+  -linters
+  -Html 
+  -css
+  -basic coding 
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Sohail Anjum**
+
+- GitHub: [@githubhandle](https://github.com/Sohail006/)
+- Twitter: [@twitterhandle](https://twitter.com/sohail1006)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/sohail-anjum-547b6bab/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Please 🙏🏽 use the acknowledgments section properly, Microverse could be a good option.